core/server/config/overrides.json

```diff
@@ -63,9 +63,6 @@
         "cannotScheduleAPostBeforeInMinutes": 2,
         "publishAPostBySchedulerToleranceInMinutes": 2
     },
-    "theme": {
-        "timezone": "Etc/UTC"
-    },
```

Comment:
Ah ok, it's in default-settings.json, rather than defaults.json :) You're probably right 👍